examples/music.py

```diff
@@ -4,11 +4,59 @@
 import discord
 from discord.ext import commands
 import lavalink
-from lavalink.audio_events import DefaultEventAdapter
+from lavalink.audio_events import TrackResumeEvent, TrackExceptionEvent, TrackStartEvent, TrackPauseEvent, TrackEndEvent, \
+    TrackStuckEvent, AudioTrack, AbstractPlayerEventAdapter
 
 time_rx = re.compile('[0-9]+')
 
 
+class DefaultEventAdapter(AbstractPlayerEventAdapter):
+    """
+    The default event adapter
+    TODO: Add more shit to this class
+    """
+
+    def __init__(self, player, ctx):
+        super().__init__(player)
+        self.ctx = ctx
+        self.bot = ctx.bot
+        self.queue = []
```

Comment:
I believe queues should be implemented within the player (or at least, the default one). Hopefully with this PR, users can customise how/where the queue is stored other than just the player.